.github/workflows/lint.yml

```diff
@@ -0,0 +1,9 @@
+name: Lint
+on: [push, pull_request]
```

Comment:
I think this check isn't running on this pull request because it's from a fork.

Reply:
For this action to run on forked pull requests, you might have to adjust the `Actions` settings.

![image](https://github.com/usernamehw/vscode-error-lens/assets/43425812/5a77c376-5984-4352-9a8d-35680c33e85d)